app/controllers/api/cloud_templates_controller.rb

```diff
@@ -8,6 +8,29 @@ def check_compliance_resource(type, id, _data = nil)
       end
     end
 
+    def import_resource(_type, _id, data = {})
+      params = %w(dst_provider_id src_provider_id src_image_id)
+      raise "Parameter 'data' has to contain non-empty values for the keys '#{params}', received: '#{data}'" if data.values_at(*params).any?(&:blank?)
```

Comment:
I'm guessing these should all be `raise BadRequestError, ""`

Reply:
`the keys '#{params}', received: '#{data}'"`

You should `#{params.join(", ")}` so this prints a little nicer

Reply:
done.